### PR TITLE
Add fingerprint for 2019~2020 Korean RAV4H_TSS2

### PR DESCRIPTION
**Pull request by cho-min**
Add Korean RAV4H_TSS2 firmware versions (2019~2020)

### CODE REVIEW

**Comment by vanillagorillaa** [comment]
Please include the dongleID from the device this was taken from

**Comment by cho-min** [comment]
My dongle is here

DongleId : c97166f4e2bb82d9

File: selfdrive/car/toyota/values.py [hunk 1]
```diff
@@ -1168,17 +1168,21 @@ class CAR:
       b'\x018966342X6000\x00\x00\x00\x00',
       b'\x01896634A25000\x00\x00\x00\x00',
       b'\x018966342W5000\x00\x00\x00\x00',
+      b'\x028966342W4000\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00',
```
Comment (adeebshihadeh) [inline selfdrive/car/toyota/values.py:1171]: which dongle ID did this version come from?

File: selfdrive/car/toyota/values.py [hunk 2]
```diff
@@ -1168,17 +1168,21 @@ class CAR:
...
       b'\x02896634A14001\x00\x00\x00\x00897CF1203001\x00\x00\x00\x00',
+      b'\x02896634A23000\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00',
       b'\x02896634A23001\x00\x00\x00\x00897CF1203001\x00\x00\x00\x00',
       b'\x02896634A14001\x00\x00\x00\x00897CF4801001\x00\x00\x00\x00',
       b'\x02896634A14101\x00\x00\x00\x00897CF4801001\x00\x00\x00\x00',
     ],
     (Ecu.esp, 0x7b0, None): [
       b'F152642291\x00\x00\x00\x00\x00\x00',
       b'F152642290\x00\x00\x00\x00\x00\x00',
+      b'F152642321\x00\x00\x00\x00\x00\x00',
```
Comment (adeebshihadeh) [inline selfdrive/car/toyota/values.py:1184]: same with this one

File: selfdrive/car/toyota/values.py (hunk 1 above)
Comment (cho-min) [reply selfdrive/car/toyota/values.py:1171]: b'\x02896634A23000\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00',

This is mine.

File: selfdrive/car/toyota/values.py (hunk 2 above)
Comment (cho-min) [reply selfdrive/car/toyota/values.py:1184]: b'F152642322\x00\x00\x00\x00\x00\x00'

This is mine.

File: selfdrive/car/toyota/values.py (hunk 1 above)
Comment (adeebshihadeh) [reply selfdrive/car/toyota/values.py:1171]: can you provide a route with these versions?

Comment (cho-min) [reply selfdrive/car/toyota/values.py:1171]: Sure, which logs do you need? Will attach it from useradmin.

**Comment by cho-min** [comment]
Changed route information to public from private.
Please check followings.
  - https://my.comma.ai/useradmin/?onebox=c97166f4e2bb82d9%7C2021-09-06--00-16-32
  - https://my.comma.ai/useradmin/?onebox=c97166f4e2bb82d9%7C2021-09-06--10-10-47
Thanks.

**Comment by adeebshihadeh** [comment]
Those two routes both have the same engine and esp versions. We also need the route the other set of engine and esp versions came from.

**Comment by cho-min** [comment]
Yes, theses routes are mine. 
Requested  another route information and dongleID to my friend.  Another one will be attached.

**Comment by cho-min** [comment]
Please add ONLY my f/w versions.
```
Ecu.engine : b'\x02896634A23000\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00'
Ecu.esp : b'F152642322\x00\x00\x00\x00\x00\x00'
```
My friend says he does it himself or next time.

Many thanks.

**Comment by adeebshihadeh** [comment]
Added yours here b670868bcd741017f04c0bc9e1dd6e123564afaa

**Pull request closed without merging**